package.json

```diff
@@ -139,6 +139,7 @@
     "tunnel-agent": "^0.6.0"
   },
   "devDependencies": {
+    "@tybys/emnapi": "^0.19.2",
```

Comment:
This added dependency should probably be omitted.

Reply:
... for now, at least :)

Reply:
Yes, that's accidental leak from rebase.

Reply:
Ha ha, this definitely doesn't give the game away ;)

Reply:
Wait for it 😛